### ISSUE TITLE
Relative import is not correctly updated when file change

### ISSUE THREAD

**Issue by AndreasArvidsson** [talonvoice/talon#448]
I have two files. tests1 imports a function from test2, but if I update the print content in test2 the print out in the log will not update until I touch tests1. 
I have a memory of a log entry somewhere along the lines of "updating  relative imports", but I don't get anything like that anymore.

version: 0.2.0-220
os: windows 10

test1.py
```
from .test2 import my_funk
cron.interval("1s", my_funk)
```
test2.py
```
def my_funk():
    print("1")
```

**Comment by lunixbochs** [comment]
are these directly in `user/`?

**Comment by AndreasArvidsson** [comment]
No they are in a subdirectory under user

**Comment by AndreasArvidsson** [comment]
I just tried putting them directly in user and it made no difference.

**Comment by lunixbochs** [comment]
fixed locally, will go out in next beta

**Comment by AndreasArvidsson** [comment]
Thank you!
I think this is the source of a very strange bug in cursorless regarding csv files not being updated correctly.

**Comment by lunixbochs** [comment]
fixed in 0.2.0-228
